printer_mods/sgr33n/300mm-x-extruder-endcap/.metadata.yml

```diff
@@ -0,0 +1,31 @@
+---
+# The title of your mod. Choose a short and descriptive name
+title: 300mm X-Extruder Endcap
+# A short description of your mod. Try to describe your mod with 1-2 short sentences
+description: This mod makes stabler and prettier the use of a 300mm profile for the X-Extruder.
+# Mod version: Set this to 1 when you are submitting a new mod. When you make changes to your mod, you may increment this number
+mod_version: 1
+# You may add your discord username here so people can contact you about your mod
+discord_username: sgr33n
+# The CAD files for your mod. Please provide them as a list.
+# Note that each mod must include at least one CAD file.
+cad:
+- CAD/300mm_X-Extruder_Endcap.step
```

Comment:
These two files are reasonable large, if you could try compressing them as a ZIP to see if we can get them down to below 2MB each that would be great.

Reply:
I've cleaned the step file of unnecessary components.